Add Kore.Attribute.Owise (#1433)
kframework/kore: docs/attributes.md

  Lbl'Unds'Set'Unds'{}(K:SortSet{},K:SortSet{}),K:SortSet{}) [idem{}()]
```

### Owise

```
owise{}()
```

> This function rule will apply only if the current configuration does not match

Reviewer: ```suggestion
This rule will apply only if the term does not unify with any other rules.
```

any other function rules for a specific symbol. Has default priority 200.

> 

Reviewer: ```suggestion
```
